tests/api2/test_108_catalog.py

```diff
@@ -0,0 +1,216 @@
+
+import os
+import pytest
+import sys
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from functions import GET, POST, DELETE, PUT, wait_on_job
+from auto_config import ha, scale, dev_test
+
+if dev_test:
+    reason = 'Skip for testing'
+else:
+    reason = 'Skipping test for HA' if ha else 'Skipping test for CORE'
+# comment pytestmark for development testing with --dev-test
+pytestmark = pytest.mark.skipif(ha or not scale or dev_test, reason=reason)
+
+official_repository = 'https://github.com/truenas/charts.git'
+truechart_repository = 'https://github.com/ericbsd/charts.git'
```

Comment:
Is this supposed to hardcode your personal repo into the test?
Don't take me wrong, you don't have to use truecharts-official, a code-frozen fork is fine for this, but I don't think it should be under https://github.com/ericbsd ;-)

Reply:
It is done in purpose for API automation testing, and I do something similar under CORE for plugins.